.github/workflows/build_test_deploy.yml

```
@@ -402,9 +402,27 @@ jobs:
       - run: npm i -g playwright-chromium@1.22.2 && npx playwright install-deps
         if: ${{needs.build.outputs.docsChange == 'nope'}}
 
-      - run: NEXT_TEST_MODE=dev node run-tests.js --type e2e --timings -g ${{ matrix.group }}/3
+      - run: npx @replayio/playwright install all
```

```suggestion
      - run: npx @replayio/playwright install chromium
```
Are we able to install just chromium here to match the above install?

---

Yep!